### ISSUE TITLE
ocp-lint doesn't detect useless spaces

### ISSUE THREAD

**Issue by OCamlPro-Iguernlala** [OCamlPro/typerex-lint#95]
If I do:

```
$ echo "   " > foo.ml
$ ocp-lint
```
The tool will emit some warnings about foo.ml:

```
Scanning files in project "."...
Found '1' file(s)
File "./foo.ml", line 1, characters 0-0:
  Warning 1: plugin_file_system/interface_missing/missing_interface
  Missing interface for './foo.ml'.
File "./foo.ml", line 1, characters 0-0:
  Warning 1: plugin_text/useless_space_line/useless_space
  Useless space or tab at end of line
Summary:
...
```

But, if I do:
```
$ echo "   " > foo.ml
$ echo "   " > foo.mli
$ ocp-lint
```

I'll not get any warning
```
Scanning files in project "."...
Found '2' file(s)
Summary:
...
```
